molecule/default/prepare.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Group hosts by OS distribution
+  hosts: all
+  tasks:
+    - name: Group hosts by OS distribution
+      group_by:
+        key: os_{{ ansible_distribution }}
+
+# The Fedora Docker images we are using
+# (geerlingguy/docker-fedora32-ansible:latest and
+# cisagov/docker-fedora33-ansible:latest) require the dnf cache to
+# be updated before they can successfully install anything.  At least,
+# they keep giving this error if I don't update the dnf cache:
+#
+# No such file or directory: '/var/cache/dnf/metadata_lock.pid'
+#
+# Related to this, I see sporadic failures when updating the DNF cache.
```

Comment:
Nice!  I have struggled with this issue for eons.

We can probably remove the dnf cache update.  I think what you did actually fixes the problem, and the cache update is unnecessary.

Reply:
Good thinking- I got rid of the cache update and cleaned up the comments a bit in https://github.com/cisagov/ansible-role-pca-gophish-composition/pull/1/commits/ee278e0b3655106959e40977b8257e59380ada14.